aws-athena/views/reporting-vw_ratio_stats.sql

```diff
@@ -1,56 +1,18 @@
 -- View containing ratios by pin, intended to feed the
 -- glue job 'reporting-ratio_stats'.
 
--- Valuation class from pardat
-WITH classes AS (
+-- Model values for corresponding triads only
+WITH model_values AS (
     SELECT
-        par.parid,
-        par.taxyr,
-        par.class,
-        CASE WHEN par.class IN ('299', '399') THEN 'CONDO'
-            WHEN par.class IN ('211', '212') THEN 'MF'
-            WHEN
-                par.class IN (
-                    '202', '203', '204', '205', '206', '207',
-                    '208', '209', '210', '234', '278', '295'
-                )
-                THEN 'SF'
-        END AS property_group
-    FROM {{ source('iasworld', 'pardat') }} AS par
-    WHERE par.cur = 'Y'
-        AND par.deactivat IS NULL
-),
-
-townships AS (
-    SELECT
-        leg.parid,
-        leg.taxyr,
-        leg.user1 AS township_code
-    FROM {{ source('iasworld', 'legdat') }} AS leg
-    WHERE leg.cur = 'Y'
-        AND leg.deactivat IS NULL
-),
-
-town_names AS (
-    SELECT
-        township_code,
-        triad_code AS triad
-    FROM {{ source('spatial', 'township') }}
-),
-
-model_values AS (
-    SELECT
-        ap.meta_pin AS parid,
-        ap.year,
+        ap.meta_pin AS pin,
+        CAST(CAST(ap.meta_year AS INT) + 1 AS VARCHAR) AS year,
```

Comment:
**issue (blocking):** Use the `year` column from `assessment_pin` and all model tables. `meta_year` is the year of characteristics, `year` is the tax year.